R/badge.R

```diff
@@ -78,6 +78,32 @@ stages <- c(
   questioning = "blue"
 )
 
+#' Create a Binder badge
+#'
+#' This prints out the markdown which will display a Binder "badge", indicating
+#' launch build your repository source codes on Binder <https://mybinder.org/>.
+#'
+#' @export
+use_binder_badge <- function() {
+
+  if (uses_git(path = proj_get()) == TRUE) {
```

Comment:
You don't need the `== TRUE`. `uses_git()` already returns either `TRUE` or `FALSE`.

Also, I wonder if this should be `uses_github()` in this case?

Reply:
Hmm. Indeed, I might as well change to `uses_github()` as you say.

[Here](https://github.com/r-lib/usethis/blob/d12b3f14c24768c65aa2abf147f1a2951c500a14/R/github.R#L72), `uses_github()` is used the same way.